src/SkillService.Validation/EditUserSkillValidator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using LT.DigitalOffice.SkillService.Data.Interfaces;
+using LT.DigitalOffice.SkillService.Models.Dto.Requests;
+using LT.DigitalOffice.SkillService.Validation.Interfaces;
+
+namespace LT.DigitalOffice.SkillService.Validation
```

Comment:
create folder for Skill, and add there create and edit validators. you can see an example in any project